partner_category_description/views/res_partner_category_views.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 Moduon Team S.L.
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0) -->
+<data>
+    <record id="view_partner_category_list" model="ir.ui.view">
```

Comment:
This record is repeated, it has the same id as inherit tree. It is not working

Reply:
I've renamed the record id so that you can search by description. It seems an copy/paste issue, but now it is fixed.